docs/concepts/storage/persistent-volumes.md

```diff
@@ -505,6 +505,7 @@ parameters:
 
 #### vSphere
 
+1. Create a persistent volume with a user specified disk format.
```

Comment:
These are not being rendered correctly as a numbered list. Indent the material between the numbered items to maintain the incremental numbering.